Soap/Client/Client.php

```diff
@@ -0,0 +1,315 @@
+<?php
+/*
+* This file is part of the CleverAge/ProcessBundle package.
+*
+* Copyright (C) 2017-2018 Clever-Age
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace CleverAge\ProcessBundle\Soap\Client;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class Client
+ *
+ * @author Madeline Veyrenc <mveyrenc@clever-age.com>
+ */
+class Client implements ClientInterface
+{
+    /**
+     * @var
+     */
+    private $code;
```

Comment:
Why not protected ? It would be nice to be able to extend this class if needed

Reply:
To force child classes to use getters and setters so it's easier to add deprecated on attribute